src/node/node_class.hpp

```diff
@@ -223,17 +223,17 @@ inline void ObjectWrap<ClassType>::setup_property(v8::Local<TargetType> target,
 
 template<typename ClassType>
 inline void ObjectWrap<ClassType>::construct(Nan::NAN_METHOD_ARGS_TYPE info) {
-    if (!info.IsConstructCall()) {
-        Nan::ThrowError("Constructor must be called with new");
-    }
     if (reinterpret_cast<void*>(s_class.constructor)) {
         auto isolate = info.GetIsolate();
         auto arguments = get_arguments(info);
         v8::Local<v8::Object> this_object = info.This();
         info.GetReturnValue().Set(this_object);
```

Comment:
I don't think so - the return value of the constructor function is disregarded in `new Foo()` and `Foo.apply(this, arguments)` expressions.

I guess I can remove this.